project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module

```diff
@@ -280,6 +494,117 @@ function c4m_message_format_group_owner_public_profile_url(Message $message) {
 }
 
 /**
+ * Resolves the id of user that executed the event.
+ *
+ * @param Message $message
+ *   The Message entity.
+ *
+ * @return int
+ *   User id.
+ */
+function c4m_message_format_event_user_id(Message $message) {
+  $wrapper = entity_metadata_wrapper('message', $message);
+  return $wrapper->field_related_user->value(array('identifier' => TRUE));
+}
+
+/**
+ * Resolves full name of user that executed the event.
+ *
+ * @param Message $message
+ *   The Message entity.
+ *
+ * @return int
+ *   User full name.
+ */
+function c4m_message_format_event_user_full_name(Message $message) {
+  $wrapper = entity_metadata_wrapper('message', $message);
+  return c4m_user_name($wrapper->field_related_user->value());
+}
+
+/**
+ * Resolves the title of attached node.
+ *
+ * @param Message $message
+ *   The Message entity.
+ *
+ * @return string
+ *   Node title.
+ */
+function c4m_message_format_node_title(Message $message) {
+  $wrapper = entity_metadata_wrapper('message', $message);
+
+  return $wrapper->field_node->label();
+}
+
+/**
+ * Resolves the type of attached node.
+ *
+ * @param Message $message
+ *   The Message entity.
+ *
+ * @return string
+ *   Node type.
+ */
+function c4m_message_format_node_type(Message $message) {
+  $wrapper = entity_metadata_wrapper('message', $message);
+
+  return $wrapper->field_node->value()->type;
+}
+
+/**
+ * Resolves the path to attached node.
+ *
+ * @param Message $message
+ *   The Message entity.
+ *
+ * @return string
+ *   Node path.
+ */
+function c4m_message_format_node_url(Message $message) {
+  $wrapper = entity_metadata_wrapper('message', $message);
+
+  return 'node/' . $wrapper->field_node->value(array('identifier' => TRUE));
+}
+
+/**
+ * Resolves number of times attached node was recommended.
+ *
+ * @param Message $message
+ *   The Message entity.
+ *
+ * @return int
+ *   Number of recommendations.
+ */
+function c4m_message_format_node_recommend_count(Message $message) {
+  $wrapper = entity_metadata_wrapper('message', $message);
+  $node =  $wrapper->field_node->value();
+
+  $property_info = array(
+    'flag_name' => 'c4m_og_content_recommend',
+  );
+  $count = c4m_helper_entity_entity_get_properties($node, array(), '', 'node', $property_info);
+
+  return empty($count) ? 0 : $count;
+}
+
+/**
+ * Resolves a teaser of nodes body.
+ *
+ * @param Message $message
+ *   The Message entity.
+ *
+ * @return string
+ *  Node body teaser.
+ */
+function c4m_message_format_node_body_teaser(Message $message) {
+  $wrapper = entity_metadata_wrapper('message', $message);
+  $node =  $wrapper->field_node->value();
+  $body_teaser = field_view_field('node', $node, 'c4m_body', 'teaser');
+
+  return strip_tags($body_teaser[0]['#markup']);
```

Comment:
Seems Html mail isn't configured properly. When it will be, you won't need to strip tags.